db/migrate/20160303183011_create_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.integer :role_id
+      t.string :role_type
+
+      t.timestamps null: false
+    end
```

Comment:
Is v1 meant to include all fields? If so, we should also add email, address, and phone number columns.